addon/services/page-title-list.js

```diff
@@ -190,6 +192,22 @@ export default Service.extend({
     }
   }),
 
+  scheduleTitleUpdate() {
+    let router = getOwner(this).lookup('router:main');
+    let routes = router._routerMicrolib || router.router;
```

Comment:
Mind adding a small inline comment here to indicate when we can remove the `|| router.router` fallback? It'll make cleanup easier when we drop support for that Ember version..

Reply:
Hmm. Travis is configured for 3.12+. Technically it could be dropped now?

We are going to so major version bump anyway, so maybe worth to drop it in one go?

Reply:
Ya, lets do that.

Reply:
Cleaned up.